### ISSUE TITLE
Does VideoSys support the "Open Sora Plan v.1.2.0" for image-to-video generation?

### ISSUE THREAD

**Issue by trouble-maker007** [NUS-HPC-AI-Lab/VideoSys#179]

**Comment by trouble-maker007** [comment]
@oahzxl thanks response, btw, is there any plan support Open Sora Plan v.1.2.0 and cogvideox, make it more efficient training, by utilizing full 3D attention, the requirement for memory storage is very high

**Comment by oahzxl** [comment]
yeah but efficient inference for now. we don't have plan to optimize their training currently.

**Comment by devtony10** [comment]
Good day, please is there any news on the image to video generation?

**Comment by oahzxl** [comment]
will be in late oct or early nov, alone with some other video enhancements

**Comment by devtony10** [comment]
Okay, thanks for your response. Could I get a workaround I can implement?

**Comment by oahzxl** [comment]
it will be so good if you'd like to implement image-to-video for open-sora-plan v1.2! if you're interested, we can create a new issue and talk about the details!

**Comment by devtony10** [comment]
I am

**Comment by devtony10** [comment]
Thanks for the offer

**Comment by oahzxl** [comment]
> Thanks for the offer

hi, i create a new issue here https://github.com/NUS-HPC-AI-Lab/VideoSys/issues/226 

look forward to your feedback
